adapters/repos/db/vector/hnsw/flat_search.go

```diff
@@ -140,6 +140,27 @@ func (h *hnsw) flatSearch(ctx context.Context, queryVector []float32, k, limit i
 	return ids, dists, nil
 }
 
+func (h *hnsw) flatMultiSearch(ctx context.Context, queryVectors [][]float32, k int,
```

Comment:
Why do we not allow the flat index to use multi-vector when we have an implementation of the flat search here?

Reply:
This can be planned as a future feature. The computation for the flat index would be really slow